src/components/FlawContributors.vue

```diff
@@ -0,0 +1,166 @@
+<script setup lang="ts">
+import { onBeforeMount, ref } from 'vue';
+import { isDefined, watchDebounced } from '@vueuse/core';
+import sanitize from 'sanitize-html';
+import LabelDiv from './widgets/LabelDiv.vue';
+import useJiraContributors from '@/composables/useJiraContributors';
+import type { ZodJiraUserPickerType } from '@/types/zodJira';
+import { createCatchHandler } from '@/composables/service-helpers';
+
+const props = defineProps<{
+  taskKey: string,
+}>();
+
+const inputRef = ref<HTMLInputElement | null>(null);
+const query = ref<string>();
+const results = ref<ZodJiraUserPickerType[]>([]);
+
+const {
+  contributors,
+  isLoadingContributors,
+  searchContributors,
+  loadJiraContributors,
+  saveContributors,
+} = useJiraContributors(props.taskKey);
+
+onBeforeMount(loadJiraContributors);
+
+watchDebounced(query, async () => {
+  if (!isDefined(query)) {
+    return;
+  }
+  const users = await searchContributors(query.value);
+  results.value = users ?? [];
+}, { debounce: 500 });
+
+
+const onFocus = () => {
+  inputRef.value?.focus();
+};
+
+
+const onBlur = (event: FocusEvent) => {
```

Comment:
Minimum relevance: should we avoid double empty lines unless necessary?
May be a new lint rule candidate?